doc/content/design/_index.md

```diff
@@ -0,0 +1,6 @@
++++
+title = "Design Documents"
+menuTitle = "Designs"
++++
+
+{{< design_docs_list >}}
```

Comment:
Sorry, will the `design_docs_list` be rendered as the list of docs under the folder? But I don't see it is rendered by the browser.

Reply:
Oh, I see it: [doc/layouts/shortcodes/design_docs_list.html](https://github.com/xapi-project/xen-api/pull/5664/files#diff-f9479e661726cb84c3528443dd06c989535f7f004fc0605a990551a58eb1933c)

Reply:
Yes, the `{{< ... >}}` bit refers to a "shortcode" in Hugo, which is something that can be included in a markdown doc and is automatically rendered as HTML.